ga4gh/serverconfig.py

```diff
@@ -26,6 +26,12 @@ class BaseConfig(object):
     SIMULATED_BACKEND_VARIANT_DENSITY = 0.5
     SIMULATED_BACKEND_NUM_VARIANT_SETS = 1
 
+    # Options for OIDC
+    # OIDC_PROVIDER = "https://accounts.google.com"
+    # OIDC_CLIENT_ID = "XXXX"
+    # OIDC_CLIENT_SECRET = "XXXX"
+    SERVER_NAME = "localhost:8001"
```

Comment:
So use an alternate file for OIDC configuration, and not serverconfig.py? I don't know about mitre, but google and other OIDC providers have a discovery endpoint that will at least give a lot of your mitre example. You should only have to configure the client id and secret, and the redirect uri?